### PR TITLE
feat: add slider

### PR DESCRIPTION
**Pull request by oSlavivna**
added a slider component.
photos automatically change on the background. 
 
added text adapters. changed the gradient on the background to match the corporate colors

### CODE REVIEW

File: src/components/common/Slider/Slider.module.scss
Comment (biryukov-m) [inline src/components/common/Slider/Slider.module.scss:1]: move this component to  src/features/home/sections/FirstViewSection/components/Slider

File: src/components/common/Slider/index.tsx
Comment (biryukov-m) [inline src/components/common/Slider/index.tsx:1]: move this component to  src/features/home/sections/FirstViewSection/components/Slider

File: src/components/common/Slider/Slider.module.scss
Comment (biryukov-m) [reply src/components/common/Slider/Slider.module.scss:1]: rename to ImageSlider

File: public/img/hero/test1.png
Comment (biryukov-m) [inline public/img/hero/test1.png:1]: export from figma .jpeg in x2 resolution

File: public/img/hero/test2.png
Comment (biryukov-m) [inline public/img/hero/test2.png:1]: reexport

**Review by biryukov-m — CHANGES_REQUESTED** [review]
- rename and move files (comments)
- use react-slick library
- enable lazy-loading for images
- export from figma .jpeg in x2 resolution

**Review by biryukov-m — DISMISSED** [review]

**Review by biryukov-m — APPROVED** [review]

**Review by OlhaPo — APPROVED** [review]

**Pull request merged**